### DISCUSSION TITLE
The puzzle for SMS 2FA auth doesn't work

### DISCUSSION THREAD

**Discussion by mauro-capannolo** [community/community#64711] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

hello,
when i try to add the SMS method for2FA authentication the puzzle, which verify the human action crashes regardless the answer.
thanks for your help

**Comment by arafajs** [comment]
Try using a different browser.

**Reply by mauro-capannolo** [reply]
Great!
thank you again!
SOLVED!

**Comment by Wouter1** [comment]
Tor browser does not show the puzzle.
I assume your puzzle uses inherently insecure methods and that tor refuses it for that reason.
